docs/topics/rest-level-0-mission-intro.adoc

```diff
@@ -1,7 +1,7 @@
 include::rest-level-0-mission-intro-paragraph.adoc[]
-More background information on REST is available in xref:about_rest[].
+For more background information on REST, see xref:about_rest[].
 
-This is an introduction to the mechanics of interacting with a remote service using the HTTP protocol. Specifically, this booster is an application that allows a user to:
+This booster introduces the mechanics of interacting with a remote service using the HTTP protocol. It allows a user to complete these tasks:
```

Comment:
Not sure what the "complete these tasks" adds. It feels weird to read "it allows to complete this task: execute an HTTP GET request". I'd much rather read "it allows to: execute an HTTP GET request".

Reply:
I would replace the sentence with _It allows you to:._

Reply:
Replaced sentence with It allows you to:  since HTTP GET request is already in the bullet list below.